docs/HowTo/Monitor/Metrics.md

```diff
@@ -53,6 +53,12 @@ To configure Prometheus and run with Teku:
      --metrics-enabled=true --metrics-categories=BEACON,PROCESS,LIBP2P,JVM,NETWORK,PROCESS
      ```
 
+    !!! important
+
+        If running Prometheus on a different host to your Teku node (any host other than `localhost`),
+        ensure you add the Prometheus host to [`--metrics-host-allowlist`](../../Reference/CLI/CLI-Syntax.md#metrics-host-allowlist)
```

Comment:
It's not the prometheus host that needs to be added, but the hostname that prometheus uses to connect to Teku.

So if prometheus is configured to get metrics from `http://teku.local:8008/metrics` then `teku.local` has to be in the metrics host allow list.  It doesn't matter what the hostname of the prometheus box is.